db/audit_schema.rb

```diff
@@ -13,6 +13,7 @@
 ActiveRecord::Schema[7.2].define(version: 2025_03_24_180818) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "postgis"
```

Comment:
Is this necessary for the columns you're adding here? I think postgis is for geospacial data

Reply:
@LindseySaari I only added this file because I was trying to get around the audit errors that I was getting on the closed PR. When I made the migration rails updated the audit table. So, idk.